setup.py

```diff
@@ -110,6 +110,27 @@
   DEFINE_MACROS += (('PyMODINIT_FUNC', pymodinit),)
 
 
+# By default, Python3 distutils enforces compatibility of
+# plugins with the OSX version Python3 was built with.
+# For Python3.4, this is OSX 10.6, but we need Thread Local Support (__thread)
+# from OSX 10.7
+def get_os_version_osx():
+  import sysconfig, re
+  platform = sysconfig.get_platform()
+  platform_match = re.match("macosx-(.*)-.*", platform)
+  if platform_match:
+    if platform_match.groups()[0] <= "10.6":
```

Comment:
This works atm simply because OS X versions start from `10.0 - Cheetah` and end at `10.11 - El Capitan`. Wonder how this works with the version numbering will be affected once they move away from OS X to just MacOS. I guess we'll find out at WWDC in a week. As for `distutils.version.StrictVersion`, it was marked for deprecation under [PEP 386](https://www.python.org/dev/peps/pep-0386/) which was superseded  by [PEP 440](https://www.python.org/dev/peps/pep-0440/) which is implemented in `parse_version` under `pkg_resources` module.